### PR TITLE
master

### PR DESCRIPTION
**Pull request by testaccount120**

### CODE REVIEW

File: src/main/java/Class002.java [hunk 1]
```diff
@@ -1,8 +1,8 @@
 package main.java;
-
+//ewwefwefewreverberbe
 public class Class002 {
     public static void main(String[] args) {
-        System.out.println("Iggg <branch> is specifiederherherh, git rebase will perform an automatic git switch <branch> before doing anything else. Otherwise it remains on the current branch.\n" +
+        System.out.println("If <branch> is specified  git rebase will perform an automatic git switch <branch> before doing anything else. Otherwise it remains on the current branch.\n" +
```
Comment (testaccount120) [inline src/main/java/Class002.java:5]: qqqqqqqqqqq

File: src/main/java/Class20.java [hunk 2]
```diff
@@ -2,4 +2,5 @@
 
 public class Class20 {
     int a;
+    //new
```
Comment (testaccount120) [inline src/main/java/Class20.java:5]: - hrthrth
-

Comment (testaccount120) [reply src/main/java/Class20.java:5]: hrthretherht

File: src/main/java/Class002.java (hunk 1 above)
Comment (veronikatimkina) [reply src/main/java/Class002.java:5]: tehrthreht

File: src/main/java/Class20.java (hunk 2 above)
Comment (testaccount120) [reply src/main/java/Class20.java:5]: rthrhthr

File: src/main/java/Class002.java (hunk 1 above)
Comment (testaccount120) [reply src/main/java/Class002.java:5]: rthrthrthrh

File: src/main/java/Class20.java (hunk 2 above)
Comment (testaccount120) [reply src/main/java/Class20.java:5]: trhrethrthr

Comment (testaccount120) [reply src/main/java/Class20.java:5]: sfbdfbdfbd

Comment (testaccount120) [reply src/main/java/Class20.java:5]: vfffffffffffff

Comment (testaccount120) [reply src/main/java/Class20.java:5]: aaaaaaaaaaaaaaaaa

**Comment by testaccount120** [comment]
single comment

File: src/main/java/Class20.java (hunk 2 above)
Comment (testaccount120) [reply src/main/java/Class20.java:5]: wrgergergerg

Comment (testaccount120) [reply src/main/java/Class20.java:5]: wegegegegeg

Comment (testaccount120) [reply src/main/java/Class20.java:5]: ewfwefwefwef

Comment (testaccount120) [reply src/main/java/Class20.java:5]: wqdqwdqwdqwd

**Review by testaccount120 — COMMENTED** [review]
dsvsdvsdvsdv

File: src/main/java/Class002.java (hunk 1 above)
Comment (testaccount120) [reply src/main/java/Class002.java:5]: -

File: src/main/java/Class20.java (hunk 2 above)
Comment (testaccount120) [reply src/main/java/Class20.java:5]: r4gt4t34t43t4

File: src/main/java/Class002.java (hunk 1 above)
Comment (testaccount120) [reply src/main/java/Class002.java:5]: iegufberig erigherihgeir geriuhgeirughiergh eirugheiurhgierug eiugheiurghieurhg eiugheiugheirg eirgheiurghei eihgierughie eighiuerhgierg eihgieurgh eiughieughieg

File: src/addFunctionWithoutDefaultValue.kts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// "Add 'abstract fun foo(x: String = "")' to 'I'" "true"
+interface I
+
+class Foo : I {
```
Comment (veronikatimkina) [inline src/addFunctionWithoutDefaultValue.kts:4]:
```suggestion
class Foo : J {
```
aaaaaaa

**Pull request merged**